website/docs/concepts/event-bridge/transform.md

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 3
+---
+
+# Transform
+
+[Event Bus Input Transforms](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-transform-target-input.html) transform events matched by [Rules](./rule).
+
+Functionless allows typescript to be used when defining an input transform.
```

Comment:
Something like this?
> Functionless translates a purely functional closure into the Event Bus Input Transform JSON format. THis enables you to implement transforms using syntax such as literals and spread expressions.

Reply:
This may be too complex and technical. Simpler way of saying it?

Reply:
Save that complexity for the syntax page?

Reply:
Are we being too redundant by repeatedly explaining how Functionless supports translating TypeScript into XYZ language? Or is it helpful to be redundant on that point?

I also worry that `transforms the closure` may be too complicated for average readers. Should we granularly explain what this means with examples?

Reply:
closure -> code.

Overall, I think its good to callout which features are being aided by compilation.

Gradually explaining, my attempt at that was

![image](https://user-images.githubusercontent.com/289213/171629296-f341cce9-a5eb-4fa8-a230-201177369833.png)